app/controllers/posts_controller.rb

```diff
@@ -9,11 +9,20 @@ def new
   end
```

Comment:
Please solve the `N+1` problem when fetching all posts and their comments for a user in the `posts#index`. You may learn more about the `n+1` problem and eager loading [here](https://dev.to/junko911/rails-n-1-queries-and-eager-loading-10eh).